ReactNativeClient/lib/ShareExtension.ts

```diff
@@ -10,4 +16,4 @@ const ext = (Platform.OS === 'android' && NativeModules.ShareExtension) ?
 		close: () => {},
 	};
 
-export default ext;
+export { ShareExtension };
```

Comment:
The `export default` was good actually. As a convention, all module should export a default member named like the filename (again I think there's an eslint for this, so I'll try to add this).

Reply:
Then you can import like so: `import ShareExtension, { SharedData } from './ShareExtension';`

Reply:
Done